### PR TITLE
sync with english version for MDN content

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for MDN content

### CODE REVIEW

File: files/zh-cn/mdn/community/mdn_content/index.md [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
-> **备注：** 我们有很多内容上的问题——非常感谢你在解决这些问题时给予的任何帮助！
+> **备注：** 我们有很多内容上的错误——非常感谢你在解决这些错误时给予的任何帮助！
 
-## 我们需要什么帮助？
+## 我们需要什么帮助
 
-为了帮助你选择要处理的内容问题，我们使用 GitHub 标签对它们进行了排序。
+为了帮助你选择要处理的内容议题，我们使用 GitHub 标签对它们进行了排序。
 
 下面的标签可以帮助你根据空闲时间来选择任务。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/mdn_content/index.md:16]:
```suggestion
下面的标签可以帮助你根据完成任务的估计工作量选择任务。
```

File: files/zh-cn/mdn/community/mdn_content/index.md [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
-| [内容仓库中花费不到 30 分钟的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-30mins%22) | 可能需要花费不到 30 分钟的任务。 |
-| [内容仓库中花费不到 3 小时的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-3hr%22)     | 可能需要花费不到 3 小时的任务。  |
-| [内容仓库中花费不到 2 天的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-2days%22)     | 可能需要花费不到 2 天的任务。    |
+| 标签                                                                       | 描述                                                                         |
+| -------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | 可能只需耗费一点精力的任务，例如修复拼写错误或添加几句用于澄清某一事实的话。 |
+| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | 可能需要付出不小的精力的任务。例如，更新页面以反映正式规范中的变更。         |
+| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | 可能需要耗费大量精力的任务，例如撰写新的指南和参考页。                       |
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/mdn_content/index.md:22]: should we use `translated-content` for these links?

File: files/zh-cn/mdn/community/mdn_content/index.md [hunk 3]
```diff
@@ -1,49 +1,47 @@
...
 - 你需要对你选择帮助的主题领域（例如 JavaScript、CSS）有一定的了解。
-- 你要帮助的大多数例子和页面都是用英语写的，所以你应该能够的理解英语。但是如果你的英语不够好，不要担心！我们的团队非常乐意帮助清理任何文字内容。
+- 你要帮助的大多数示例和页面都是用英语写的，所以你应该能够理解英语。但是如果你的英语不够好，不要担心！我们的团队非常乐意帮助清理任何文字内容。
 
-## 你要如何帮助？
+## 你要如何帮助
 
-1. 首先，如果你还没有 [GitHub 帐户](https://github.com/join)，你需要注册一个。你需要通过它来沟通 GitHub 上的问题。
-2. 接下来，选择一个或多个您想帮助的主题领域，并使用上面的列表来帮助您进行选择。如果您不确定什么是好的选择，请在 [Matrix](https://wiki.mozilla.org/Matrix) 的[MDN Web 文档聊天室](https://chat.mozilla.org/#/room/#mdn:mozilla.org)寻求帮助。
+1. 首先，如果你还没有 [GitHub 帐户](https://github.com/join)，你需要注册一个。你需要通过它来就 GitHub 上的议题进行交流。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/mdn_content/index.md:39]: which is more used in MDN - "账户" or "帐户" ?🤔

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/mdn/community/mdn_content/index.md (hunk 3 above)
Comment (yin1999) [reply files/zh-cn/mdn/community/mdn_content/index.md:39]: Yes. We should use "账户"

```suggestion
1. 首先，如果你还没有 [GitHub 账户](https://github.com/join)，你需要注册一个。你需要通过它来就 GitHub 上的议题进行交流。
```

File: files/zh-cn/mdn/community/mdn_content/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/mdn/community/mdn_content/index.md:22]: If we refer to `translated-content`, we have to modify most content in this document. As we won't label issue by technologies.

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**